### PR TITLE
Add support for registering consumer at specific Message ID

### LINKED ISSUE #6: Support creating groups from end of stream, not just beginning
Because the message ID we're specifying when creating the group is always `0`, consumers will need to process all messages in the stream before getting to the latest message. For a system I'm building, I'm looking to have the semantics be more lossy when a new consumer comes to life.

Would it be possible to extend the API to include a way to specify whether we want all messages, or only new messages.

Here is the relevant line of code:

https://github.com/robinjoseph08/redisqueue/blob/d95817e5c69fb071b4cdb2232235854f872c7926/consumer.go#L156

### PR DESCRIPTION
**Pull request by theckman**
Some consumers may not want to process old messages in the stream, and to
instead only start consuming from the latest. This adds a RegisterWithID
function that allows you to specify any message ID. Per the Redis documention,
`0` means the oldest message in the stream while `$` means the latest.

Fixes #6

### CODE REVIEW

**Comment by theckman** [comment]
@robinjoseph08 might you have a moment to 👀 and :shipit: into a tagged release?

File: consumer.go [hunk 1]
```diff
@@ -118,25 +123,39 @@ func NewConsumerWithOptions(options *ConsumerOptions) (*Consumer, error) {
...
 
 		stopReclaim: make(chan struct{}, 1),
 		stopPoll:    make(chan struct{}, 1),
 		stopWorkers: make(chan struct{}, options.Concurrency),
 	}, nil
 }
 
+// RegisterWithLastID is the same as Register, except that it also lets you
+// specify the oldest message to receive. This can be any valid message ID, "0"
+// for all messages in the stream, or "$" for only new messages.
```
Comment (robinjoseph08) [inline consumer.go:141]: Let's add a note here saying that the `id` value is only used when the consumer group is created for the first time, and if the consumer group already exists, the value of `id` will be ignored.

Comment (theckman) [reply consumer.go]: @robinjoseph08 I've pushed an amended commit which I hope reads okay.

**Comment by theckman** [comment]
I raised another PR while I have your 👀. 😄

**Pull request merged**